### DISCUSSION TITLE
Planner e orario docente settimanale

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#347] [Idee e proposte]
Salve,
sarebbe possibile quando si schiaccia su Registro con un profilo docente, al posto di avere la lista con le proprie classi assegnate e le altre classi dell'istituto se si potrebbe avere una tabella con le ore previste per quella giornata/settimana come troviamo in altri registi (allego foto n.1) e se si potrebbe creare una sezione dove ogni docente può aggiungere il proprio orario (allego foto n.2 e 3), che poi verrà visualizzato tramite la tabella della foto n.1.
![2A0EB4F0-38AA-4487-A2C3-946ABFA5CEC1](https://user-images.githubusercontent.com/96265220/189892616-9f8d87b3-8536-4711-af7b-818e9a3c9cab.jpeg)
![91BEACD6-9A0F-45EF-885F-684D0E2309CE](https://user-images.githubusercontent.com/96265220/189892629-bf1cd843-8a74-4340-bf89-02d53d02a64e.jpeg)
![12D520EA-8562-4395-9E6D-ABFFFDFD58BA](https://user-images.githubusercontent.com/96265220/189892636-9efc6edd-f4b9-44d7-9064-f8a4b1b4d836.jpeg)

**Comment by trinko** [comment]
La segno come richiesta.
